fre/make/run_fremake_script.py

```diff
@@ -1,14 +1,9 @@
-#!/usr/bin/python3
 '''
-date 2023
-author(s): Tom Robinson, Dana Singh, Bennett Chang
-fre make is used to create, run and checkout code, and compile a model.
+Create, run, checkout code, and compile a model to create a model executable.
```

Comment:
This seems unclear, maybe `Creates script to checkout the model code and to compile`?